lib/evaluate/intrinsics.cc

```diff
@@ -198,6 +200,8 @@ static constexpr IntrinsicDummyArgument MatchingDefaultKIND{"kind",
 static constexpr IntrinsicDummyArgument SubscriptDefaultKIND{"kind",
     {IntType, KindCode::kindArg}, Rank::scalar,
     Optionality::defaultsToSubscriptKind};
+static constexpr IntrinsicDummyArgument RequiredDIM{
+    "dim", {IntType, KindCode::dimArg}, Rank::scalar};
```

Comment:
Thanks.  My "build everywhere with everything" script must have a bug with exit codes, since I see the same failure in the logs but the overall script passed.  I'll fix it up.

Reply:
Build problems have been resolved.